OpenSim/Common/DataTable.h

```diff
@@ -997,6 +998,82 @@ class DataTable_ : public AbstractDataTable {
         _indData[rowIndex] = value;
     }
 
+    /** Reorder columns of the DataTable by providing the new order of 
+    column indices. The new order is basically a permutation of column indices
+    0 to num-columns-1. For example:
+    \code
+    // Change order of last three columns only. Leave the first two columns 
+    // as is.
+    table.reorderColumns({0, 1, 5, 3, 4});
```

Comment:
Your example code will throw a `ColumnIndexOutOfRange` exception. The argument should be `{0, 1, 4, 2, 3}`.